src/components/articleId/SingleArticle.jsx

```diff
@@ -22,6 +25,21 @@ const SingleArticle = () => {
       });
   }, [article_id]);
 
+  const errReset = () => {
+    setErr(null);
+  };
+
+  const handleVote = (event) => {
```

Comment:
You use `handleVote` for the single article component as well as the comment card. Might be best to extract this and make it a reusable function so you can use it for both of the components without writing it twice!

Reply:
This has been done, handleVote is now in a new utils.js file